docs/articles_en/get-started/install-openvino/install-openvino-conda.rst

```diff
@@ -94,6 +94,14 @@ To reactivate your Conda environment, execute the following command:
 Once you have reactivated your Conda environment, make sure that all the necessary environment
 variables are properly set and proceed with linking the OpenVINO libraries.
 
+Enabling GPU device for inference
++++++++++++++++++++++++++++++++++
+
+To use a GPU device for OpenVINO inference, you must install OpenCL ICD:
```

Comment:
Let's mention that it's Linux only and on Windows it's not required to do anything, because OpenCL is installed with GPU driver.

Reply:
Have added info with https://github.com/openvinotoolkit/openvino/pull/24390/commits/a5d4ae6650c2e10f34c923cc878dbff6165c9e54